src/index.html

```diff
@@ -6,9 +6,29 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="Moyo logo">
+      </a>
+      <nav>
+        <ul>
+          <li><a href="#" class="is-active">Apple</a></li>
+          <li><a href="#">Samsung</a></li>
+          <li><a href="#">Smartphones</a></li>
+          <li data-qa="hover"><a href="#">Laptops & Computers</a></li>
```

Comment:
and please apply code styling recommendations from a checklist
https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md